examples/chip-tool/BUILD.gn

```diff
@@ -21,7 +21,7 @@ import("${chip_root}/src/app/chip_data_model.gni")
 assert(chip_build_tools)
 
 chip_data_model("data_model") {
-  cluster_sources = [ "media-playback-client" ]
+  cluster_sources = [ "thermostat-client" ]
```

Comment:
Is this just a bad diff? I assume we don't mean to replace this?

Reply:
No it's not I want to replace it. There's no need for media playback client code as it's just sending response. 
What I haven't knew at the time is that if you name your cluster commads properly you get that for free.
Server command name: **Play**
Client command name: **Play**Response

If every client command has the Response string attached to a name you don't need client code